### PR TITLE
typescript configuration

### PR DESCRIPTION
**Pull request by Carlosches**

### CODE REVIEW

File: cypress/integration/google.spec.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+describe('This is my first cypress test', () => {
+    it('should have a title', () => {
+        cy.visit('https://www.google.com/');
+        cy.title().should('eq', 'Google');
+    });
+});
```
Comment (garzuzo) [inline cypress/integration/google.spec.ts]: add empty line at the end of the file

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+            "cypress"
+        ]
+    },
+    "include": [
+        "cypress/**/*.ts"
+    ],
+    "exclude": [
+        "node_modules/"
+    ]
+}
```
Comment (garzuzo) [inline tsconfig.json]: add empty line at the end of the file

File: cypress/integration/google.spec.ts [hunk 3]
```diff
@@ -3,4 +3,4 @@ describe('This is my first cypress test', () => {
         cy.visit('https://www.google.com/');
         cy.title().should('eq', 'Google');
     });
-});
\ No newline at end of file
+});
```
Comment (Carlosches) [inline cypress/integration/google.spec.ts:6]: In the local file is the empty line at the end

File: cypress/plugins/index.js [hunk 4]
```diff
@@ -16,7 +16,18 @@
...
 module.exports = (on, config) => {
-  // `on` is used to hook into various events Cypress emits
-  // `config` is the resolved Cypress config
+  const options = {
+    webpackOptions: require('../../webpack.config'),
+  }
+
+  on('file:preprocessor', wp(options))
 }
+
```
Comment (holgiosalos) [inline cypress/plugins/index.js]: This file has two empty lines, just is needed one and only one :).

File: cypress/integration/google.spec.ts (hunk 3 above)
Comment (holgiosalos) [reply cypress/integration/google.spec.ts:6]: you're right

File: webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+              loader: 'ts-loader',
+              options: {
+                transpileOnly: true,
+              },
+            },
+          ],
+        },
+      ],
+    },
+  }
```
Comment (holgiosalos) [inline webpack.config.js:24]: Fix the indentation of the brackets, they should go in the same column of the opening bracket

Comment (holgiosalos) [reply webpack.config.js:24]: nevermind... the identation is ok

**Pull request merged**